### PR TITLE
Issue #2455 - update "as" operation to match "as" function

### PR DESCRIPTION
**Pull request by JohnTimm**
Signed-off-by: John T.E. Timm <johntimm@us.ibm.com>

### CODE REVIEW

**Review by prb112 — APPROVED** [review]
LGTM, do we have a test to test the new behavior?

**Review by prb112 — APPROVED** [review]
LGTM

File: fhir-path/src/test/java/com/ibm/fhir/path/test/FHIRPathAsTest.java [hunk 1]
```diff
@@ -38,6 +39,18 @@ void testAsOperation() throws Exception {
         assertEquals(result.size(), 0, "Number of selected nodes");
     }
 
+    @Test
+    void testAsOperationSystemValue() throws Exception {
```
Comment (prb112) [inline fhir-path/src/test/java/com/ibm/fhir/path/test/FHIRPathAsTest.java:43]:
```suggestion
    void testAsOperationSystemValue() throws Exception {
        // Testing 'as'
```

**Review by lmsurpre — APPROVED** [review]
LGTM

**Pull request merged**